github_pr_to_internal_pr/Dockerfile

```diff
@@ -12,6 +12,8 @@ RUN apt-get update && \
     pip3 install --upgrade pip && \
     pip3 install -r /tmp/requirements.txt
 
+RUN chown -R $(id -u):$(id -g) $PWD
```

Comment:
Please add a comment here explaining the reasoning, also any pointers (per PR description).

In the long term, we should fix this in the workflow itself by git config `safe.directory` approach.

Reply:
Done